config/config.go

```diff
@@ -123,6 +130,19 @@ type Config struct {
 	EnableDynamicConfig bool `toml:"enable-dynamic-config" json:"enable-dynamic-config"`
 }
 
+// UpdateTempStoragePath is to update the `TempStoragePath` if port/statusPort was changed
+// and the `tmp-storage-path` was not specified in the conf.toml or was specified the same as the default value.
+func (c *Config) UpdateTempStoragePath() {
+	if c.TempStoragePath == tempStorageDirName {
```

Comment:
I think this check is redundant.

Reply:
If the TempStoragePath was specified in the config file, UndateTempStoragePath() will override the customized `tmp-storage-path` value without this check. Am I right?

Reply:
Make sense. Thanks for your explanation.